config/default.yml

```
@@ -194,6 +194,11 @@ Minitest/TestMethodName:
   Enabled: 'pending'
   VersionAdded: '0.10'
 
+Minitest/UnreachableAssertion:
+  Description: 'This cop check for an `assert_raises` block containing any unreaclable assertions.'
```

```suggestion
  Description: 'This cop checks for an `assert_raises` block containing any unreachable assertions.'
```
(sorry, missed the 's' in my prior comment.)